master/internal/api_experiment.go

```diff
@@ -245,7 +245,7 @@ func (a *apiServer) GetExperiments(
 		apiv1.GetExperimentsRequest_SORT_BY_END_TIME:    "end_time",
 		apiv1.GetExperimentsRequest_SORT_BY_STATE:       "state",
 		apiv1.GetExperimentsRequest_SORT_BY_NUM_TRIALS:  "num_trials",
-		apiv1.GetExperimentsRequest_SORT_BY_PROGRESS:    "progress",
+		apiv1.GetExperimentsRequest_SORT_BY_PROGRESS:    "COALESCE(progress, 0)",
```

Comment:
wouldnt we want coalesce(progress, 1)? seems like nulls is closer to 1 than 0

Reply:
We are now setting progress to 1 on completed experiments so I think this is now a moot point
but at the time, I believe the both unstarted and completed experiments could have progress=NULL and wanted them to follow the existing sort behavior

Reply:
ah, yeah. i would also call that very strange, that we create them with NULL not 0 progress. idk.